test/membrane_live_web/channels/event_channel_test.exs

```diff
@@ -16,73 +18,49 @@ defmodule MembraneLiveWeb.EventChannelTest do
         title: "Test webinar"
       })
```

Comment:
IMO this shouldn't be possible to add a Webinar without a moderator.

Reply:
Will be solved in https://membraneframework.atlassian.net/jira/software/c/projects/LIVE/boards/11?modal=detail&selectedIssue=LIVE-94